src/components/tx/DecodedTx/index.tsx

```diff
@@ -37,9 +39,14 @@ const DecodedTx = ({ tx, txId }: DecodedTxProps): ReactElement | null => {
     return null
   }
 
+  const onChangeExpand = () => {
+    setIsAccordionExpanded((prev) => !prev)
+    trackEvent({ ...MODALS_EVENTS.TX_DETAILS, label: isAccordionExpanded ? 'Close' : 'Open' })
+  }
+
   return (
     <Box mb={2}>
-      <Accordion elevation={0}>
+      <Accordion elevation={0} onChange={onChangeExpand}>
```

Comment:
Maybe we should set the the `expanded` prop of the `Accordion` to `isAccordionExpanded. What do you think?

Reply:
Why? It seems to work regardless.